app/components/support_interface/ucas_match_table_component.html.erb

```diff
@@ -14,7 +14,26 @@
     <% table_rows.each do |table_row| %>
       <tr class="govuk-table__row">
         <td class="govuk-table__cell"><%= table_row[:course_code] %></td>
-        <td class="govuk-table__cell"><%= table_row[:course_details] %></td>
+        <td class="govuk-table__cell">
+          <%= table_row[:course_details] %>
+          <% if table_row[:provider_contacts].any? %>
+            <details class="govuk-details" data-module="govuk-details">
+              <summary class="govuk-details__summary">
+                <span class="govuk-details__summary-text">
+                  Contact details
+                </span>
+              </summary>
+              <div class="govuk-details__text">
+                <% table_row[:provider_contacts].each do |provider_contact| %>
+                  <%= render ProviderInterface::UserListCardComponent.new(
```

Comment:
Now thinking about it, I don't think we should be using this component. Since the link goes to the provider interface, which will show the signed in support users "Access denied". 
Maybe we should just make it a really simple like a comma separated list the email addresses.

Reply:
@AgaDufrat Yes that makes sense, I am not very familiar with persmissions so I missed that. 

I do however think  that displaying a similar formatted list would make sense so they have both the name as well as the email of the person they are contacting. Maybe a non-bulleted list. I'll check the UI GDS components and see what fits best.